modules/core/src/main/scala/fs2/kafka/internal/WithAdminClient.scala

```diff
@@ -20,34 +20,23 @@ private[kafka] sealed abstract class WithAdminClient[F[_]] {
 private[kafka] object WithAdminClient {
   def apply[F[_]](
     settings: AdminClientSettings[F]
-  )(
-    implicit F: Concurrent[F],
-    context: ContextShift[F]
-  ): Resource[F, WithAdminClient[F]] = {
-    val blockerResource =
-      settings.blocker
-        .map(Resource.pure[F, Blocker])
-        .getOrElse(Blockers.adminClient)
-
-    blockerResource.flatMap { blocker =>
+  )(implicit F: Async[F]): Resource[F, WithAdminClient[F]] = {
       Resource[F, WithAdminClient[F]] {
         settings.createAdminClient.map { adminClient =>
           val withAdminClient =
             new WithAdminClient[F] {
               override def apply[A](f: AdminClient => KafkaFuture[A]): F[A] =
-                context.blockOn(blocker) {
-                  F.suspend(f(adminClient).cancelable)
-                }
+                ???
+                // context.blockOn(blocker) {
+                //   F.suspend(f(adminClient).cancelable)
```

Comment:
mm interesting. Might need to use `blocking` first, and then convert the `Future`, but needs more thought

Reply:
Yeah, need to to grok what's going on with `cancelable`.

Reply:
Gave this a try with 50562c6 - I'm not at all sure whether it's correct though.